doc/getting-started/gsg_quick_setup.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+# Copyright (C) 2018 Intel Corporation.
+# SPDX-License-Identifier: BSD-3-Clause
+# This script is aim to setup or upgrade sos & uos automatically.
+
+# switch for download uos function
+skip_download_uos=0
+# turn on if you need to setup uos
+upgrade_uos=0
+
+function print_help()
+{
+echo "Usage:"
+echo "Launch this script as: ./flash.sh -t 28100"
+echo "-t to specify clear linux version for upgrading"
+echo "-p to specify a proxy server"
+echo "-m to use swupd mirror url"
+echo "-u to upgrade uos, if you just upgrade sos using this script, you'd better to manually reboot sos then continue to use this function."
+echo "-s, skip downloading uos; if enabled, you have to download uos img firstly before upgrading, default is off"
+exit 1
+}
+
+function upgrade_sos()
+{
+# setup mirror, proxy env if specified parameters to p & m opt.
+swupd mirror -u
+if [[ -n $mirror ]]; then
+    echo "Add swupd mirror..."
+    swupd mirror -s $mirror
+fi
+unset https_proxy
+if [[ -n $proxy ]]; then
+    echo "Add proxy..."
+    export https_proxy=$proxy
+fi
+echo "disable auto update..."
+swupd autoupdate --disable
+
+# Compare with current clear linux and skip upgrade sos if get the same version.
+if [[ $(echo -e `swupd info` | cut -d' ' -f3) -eq $target_version ]]; then
+    echo "Specified to the same clear linux version, continue to setup sos env."
+else
+    echo "Upgrade clear linux version to : $target_version ..."
+    swupd verify --fix --picky -m $target_version
+fi
+
+# Do the setups if previous process succeed.
+if [[ $? -eq 0 ]]; then
```

Comment:
Do we need to reboot after the `swupd verify fix` operation or is it not required.

Reply:
Yes, it will do reboot after sos setup is complete.